MonkeyLoader/Configuration/ConfigKeyDefault.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace MonkeyLoader.Configuration
+{
+    /// <summary>
+    /// Contains some <see cref="IConfigKeyDefault{T}"/> presets.
+    /// </summary>
+    public static class ConfigKeyDefault
+    {
+        /// <summary>
+        /// Uses the provided <see langword="unmanaged"/> value as a default.
+        /// </summary>
+        /// <typeparam name="T">The type of the config item's value.</typeparam>
+        /// <param name="value">The constant value to use as a default.</param>
+        /// <returns>A new default component.</returns>
+        public static IConfigKeyDefault<T> Constant<T>(T value) where T : unmanaged
+            => new ConfigKeyDefault<T>(() => value);
```

Comment:
I've no clue what the unmanaged here means.... and also personally still very much dislike needing lambdas for everything :stuck_out_tongue:

Reply:
Similar to struct, but with no reference types used anywhere, so with everything being copied when used (so there's no funny shared state).

Had to look that one up too.

Reply:
But yes, this could be implemented with a second type that implements the interface